device/tt_silicon_driver.cpp

```diff
@@ -1369,28 +1369,33 @@ void tt_SiliconDevice::set_fallback_tlb_ordering_mode(const std::string& fallbac
 }
 
 // TT<->TT P2P support removed in favor of increased Host memory.
+// TODO: this is in the wrong place, it should be in the PCIDevice.
```

Comment:
I plan to move around stuff like this from tt_SiliconDevice to PCIDevice and newly formed TTDevice.
Just fyi